hathor/indexes/address_index.py

```diff
@@ -92,7 +92,7 @@ def get_from_address(self, address: str) -> list[bytes]:
         raise NotImplementedError
 
     @abstractmethod
-    def get_sorted_from_address(self, address: str) -> list[bytes]:
+    def get_sorted_from_address(self, address: str, tx: Optional[BaseTransaction] = None) -> Iterable[bytes]:
```

Comment:
Maybe add info about what is this new argument in the docstring? Or maybe a more expressive name than `tx`

Reply:
Maybe `after` or `after_tx`?

Reply:
I changed to `tx_start` the parameter name and improved the docstring. I decided not to use `after`/`after_tx` because we don't return the iterator after it, we return including it.

What do you think? https://github.com/HathorNetwork/hathor-core/pull/978/commits/57558a502daa480f03a942e50f8c5861fc676ff1

Reply:
Works for me! Thanks